entity-api-spec.yaml

```diff
@@ -1,396 +1,1267 @@
+        '500':
           description: Internal error
-
-  /entities/multiple-samples:
+  '/collections/{collection_uuid}}/add-datasets':
     put:
-      summary: 'Update properties of multiple samples via one API request'
-      responses:
-        "200":
-          description: successful operation
-        "400":
-          description: Invalid json input
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "404":
-          description: The target entity could not be found
-        "500":
-          description: Internal error
-
-  /collections/{collection_uuid}}/add-datasets:
-    put:
-      summary: 'Link the given list of datasets to the target collection'
+      summary: Link the given list of datasets to the target collection
       parameters:
         - name: collection_uuid
           in: path
-          description: 'The UUID of target collection'
+          description: The UUID of target collection
           required: true
           schema:
             type: string
       responses:
-        "200":
+        '200':
           description: successful operation
-        "400":
+        '400':
           description: Invalid or misformatted collection uuid
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "404":
+        '401':
+          description: The user's token has expired or the user did not supply a valid token
+        '403':
+          description: The user is not authorized to add datasets to the collection
+        '404':
           description: The target entity could not be found
-        "500":
+        '500':
           description: Internal error
-          
-  /collection/redirect/{id}:
-    put:
-      summary: 'Redirect a request from a doi service for a collection of data'
+  '/doi/redirect/{id}':
+    get:
+      summary: Redirect a request from a doi service  
       parameters:
         - name: id
           in: path
-          description: 'The HuBMAP ID (e.g. HBM123.ABCD.456) or UUID of the target collection'
+          description: The HuBMAP ID (e.g. HBM123.ABCD.456) or UUID of the target collection
           required: true
           schema:
             type: string
       responses:
-        "200":
-          description: successful operation
-        "400":
+        '307':
+          description: Redirect to the entities landing page
+        '400':
           description: Invalid or misformatted collection identifier
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "404":
-          description: The target entity could not be found
-        "500":
+        '500':
           description: Internal error
-          
-  /entities/dataset/globus-url/{id}:
-    put:
-      summary: 'Get the Globus URL to the given dataset'
+  '/entities/dataset/globus-url/{id}':
```

Comment:
The current entity-api also supports the old endpoint `/entities/dataset/globus-url/<id>`

https://github.com/hubmapconsortium/entity-api/blob/test-release/src/app.py#L1733

Should we add it to the API docs too?

Reply:
No, let's leave this one out of the docs.  The old one will be "deprecated forever".